packages/cuba-rest-js/src/security.ts

```diff
@@ -65,13 +65,41 @@ export function isOperationAllowed(entityName: string,
   explicitPerm = entities.find(perm => perm.target === `${entityName}:*`);
   if (explicitPerm != null) return explicitPerm.value === 1;
 
+  // find entity wildcard perm match '*:read'
+  explicitPerm = entities.find(perm => perm.target === `*:${operation}`);
+  if (explicitPerm != null) return explicitPerm.value === 1;
+
   // find pure wildcard perm match '*:*'
   explicitPerm = entities.find(perm => perm.target === `*:*`);
   if (explicitPerm != null) return explicitPerm.value === 1;
 
   return perms.undefinedPermissionPolicy === 'ALLOW';
 }
 
+export function isSpecificPermissionGranted(permTarget: string, perms?: EffectivePermsInfo): boolean {
```

Comment:
`is*PermissionGranted` looks more accurate but maybe its better use `Allowed` suffix in terms of uniformity with already existing `isOperationAllowed()` ?